app/views/questionnaire.py

```diff
@@ -144,8 +143,9 @@ def post_interstitial(eq_id, form_type, collection_id, block_id):
         raise NotFound
 
     metadata = get_metadata(current_user)
-    logger.info("Redirecting user to next location %s with tx_id=%s", str(next_location), metadata["tx_id"])
-    return redirect(next_location.url(metadata))
+    next_location_url = next_location.url(metadata)
+    logger.debug("Redirecting user", redirect=next_location_url)
```

Comment:
It would be good to start working towards consistency in logging messages. I think a message of `Redirecting` and `url=next_location_url` is as clear and could be used anywhere we redirect.